storageFormats/eventMetadata.h

```diff
@@ -112,6 +115,31 @@ namespace rpwa {
 		return metadata.print(out);
 	}
 
+
+	/// \brief Handles additional saved variables of event data
+	class additionalVariables {
+	public:
+		additionalVariables() {}
+		additionalVariables(const additionalVariables&) = delete;
+		additionalVariables& operator = (const additionalVariables&) = delete;
+		double operator [](const std::string& label) const {return _additionalSavedVariables.at(label);}
+
+		bool setBranchAddresses(const eventMetadata& metaData);
```

Comment:
Add description:
reads list of tree variables from provided metaData and connects them to tree branches